bin/controller.keys.js

```diff
@@ -27,6 +27,8 @@ var _ = require('lodash');
 
 module.exports.updateKeys = function updateKeys(options, taskCallback) {
 
+    let roles = process.env.ALLOW_SSH_ACCESS_ROLES || "admin,maintain,write";
```

Comment:
@kavaribes, we probably need some kind of validation here. I mean to prevent passing unexpected values to `process.env.ALLOW_SSH_ACCESS_ROLES`. 

Also, as an improvement, I would move the default configuration to the package config or even better to YAML config.